kubernetes-aws-auth.tf

```diff
@@ -30,7 +30,7 @@ locals {
   )
 }
 
-resource "kubernetes_config_map" "aws_auth" {
+resource "kubernetes_config_map_v1" "aws_auth" {
```

Comment:
I don't think it's safe to remove aws-auth configmap. 

I've another approach for this resource in #66